flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/JoinITCase.scala

```diff
@@ -1333,7 +1335,7 @@ class JoinITCase(expectedJoinType: JoinType) extends BatchTestBase {
 }
 
 object JoinITCase {
-  @Parameterized.Parameters(name = "{0}")
+  @Parameters(name = "{0}")
```

Comment:
```suggestion
  @Parameters(name = "expectedJoinType = {0}")
```

Reply:
If the "name" is modified to "expectedJoinType". It can't be compiled successfully.

Reply:
If the "name" is modified to "expectedJoinType". It can't be compiled successfully.

Reply:
I don't quite understand why modifying the strings（value of name) doesn't go through compilation? What you should modify is the value of name, not itself.

Reply:
Sorry for the misunderstanding. Fixed.